ember-cli-build.js

```diff
@@ -5,7 +5,11 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
-    // Add options here
+    babel: {
+      plugins: [
+        'transform-decorators-legacy'
```

Comment:
@buschtoens also broke out the babel plugins to @ember-decorators/babel-transforms, you should be able to install that to get transforms set up properly